atef/widgets/config/page.py

```diff
@@ -965,21 +1037,24 @@ class PVConfigurationPage(DesignerDisplay, PageWidget):
     pv_widget_placeholder: QWidget
     pv_configuration_widget: PVConfigurationWidget
 
-    comparisons_table: QTableWidget
+    comparisons_table_placeholder: QWidget
+    comparisons_table: PagedTableWidget
     add_comparison_button: QPushButton
 
-    attr_selector_cache: WeakSet[QComboBox]
-
     def __init__(self, data: PVConfiguration, **kwargs):
         super().__init__(data=data, **kwargs)
-        # Create the static sub-widgets and place them
-        self.attr_selector_cache = WeakSet()
         self.setup_name_desc_tags_init()
         self.pv_configuration_widget = PVConfigurationWidget(data=data)
         self.insert_widget(
             self.pv_configuration_widget,
             self.pv_widget_placeholder,
         )
+
+        self.comparisons_table = PagedTableWidget(title="Comparisons")
+        self.insert_widget(
+            self.comparisons_table,
+            self.comparisons_table_placeholder
+        )
```

Comment:
You should know that you can actually promote widgets to python widgets inside the ui file using designer, removing the need to have a replaceable placeholder widget. It seems to work here though so no real need to refactor. (I also forget the precise magic for doing this)

Reply:
Oh interesting.  I'll poke around.

Reply:
Done, cleans up the python classes a bit 👍